libcontainer/integration/bench_test.go

```diff
@@ -0,0 +1,51 @@
+package integration
+
+import (
+	"os"
+	"testing"
+
+	"github.com/opencontainers/runc/libcontainer"
+)
+
+func BenchmarkExecTrue(b *testing.B) {
+	config := newTemplateConfig(b, nil)
+	container, err := newContainer(b, config)
+	ok(b, err)
+	defer destroyContainer(container)
+
+	// Execute a first process in the container
+	stdinR, stdinW, err := os.Pipe()
+	ok(b, err)
+	process := &libcontainer.Process{
+		Cwd:   "/",
+		Args:  []string{"cat"},
+		Env:   standardEnvironment,
+		Stdin: stdinR,
+		Init:  true,
+	}
+	err = container.Run(process)
+	_ = stdinR.Close()
+	defer func() {
+		_ = stdinW.Close()
+		if _, err := process.Wait(); err != nil {
+			b.Log(err)
+		}
+	}()
+	ok(b, err)
```

Comment:
Yes, in here we don't care if `cat` is running already, we just need a container in which we can exec true a bazillion times.

Unrelated to this, but I guess we use `cat` here (and in other places) because it gives us a simple way to have an init process which waits *(for stdin to close) and then exits cleanly. One alternative would be `sleep infinity` -- no need for stdin pipe, but it needs to be killed, which results in a non-zero exit code and is maybe slightly less perfect because of that.

Reply:
Makes sense. I expected we didn't care about that race :)